src/components/PointTooltip.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+
+const DEFAULT_IMAGE_URL = "https://upload.wikimedia.org/wikipedia/commons/thumb/7/79/2010-brown-bear.jpg/200px-2010-brown-bear.jpg";
+
+const timeToString = time => {
```

Comment:
Mm, we have a similar function in the ListCard component I think (might be in my open PR). Once merged, maybe we can pull this function into a util.js file so we can use it globally?